src/tests/test_filesystem.py

```diff
@@ -28,23 +28,23 @@ def test_idbfs_persist_code(selenium_standalone):
         pyodide.FS.mount(pyodide.FS.filesystems.{fstype}, {{root : "."}}, mountDir);
         """
     )
-    # create file in mount
-    selenium.run_js(
-        f"""
-        pyodide.runPython(`
-            import pathlib
-            p = pathlib.Path('{mount_dir}/test_idbfs/__init__.py')
-            p.parent.mkdir(exist_ok=True, parents=True)
-            p.write_text("def test(): return 7")
-            from importlib import invalidate_caches
-            invalidate_caches()
-            import sys
-            sys.path.append('{mount_dir}')
-            from test_idbfs import test
-            assert test() == 7
-        `);
-        """
-    )
+
+    @run_in_pyodide
+    def create_test_file(selenium_module, mount_path="/mount_test"):
```

Comment:
Why do you give `mount_path`  a default value? It's never used. Also, you can keep the name as `mount_dir`.